include/freerdp/client/rdpgfx.h

```diff
@@ -117,6 +125,12 @@ struct _rdpgfx_client_context
 	pcRdpgfxSetCacheSlotData SetCacheSlotData;
 	pcRdpgfxGetCacheSlotData GetCacheSlotData;
 
+	/* proxy callbacks */
```

Comment:
Are the callbacks specific to the proxy?

Reply:
right now yea